BookReader/BookReader.js

```diff
@@ -1720,11 +1720,6 @@ BookReader.prototype.exitFullScreen = function() {
 
     $(document).unbind('keyup', this._fullscreenCloseHandler);
 
-    var windowWidth = $(window).width();
-    if (windowWidth <= this.onePageMinBreakpoint) {
-        this.switchMode(this.constMode2up);
-    }
-
```

Comment:
QUESTION: This is such a clear, specific switch. I wonder what context led to its inclusion. CC: @rchrd2

Reply:
Hi @bfalling. Yes there was a reason for this. It is this way, so that on mobile devices, on the details pages, and not in full screen, we show the little 2 page book. And then when you go to full screen you view the 1 page mode. -- The scrolling 1 page mode on the details page is not great, because you are scrolling within a div, and it's not very usable. The idea was to show a small overview of the book, but then require the the user to go to full screen to actually read the book.

You can see the original designs here: https://docs.google.com/presentation/d/1b72JUiW2ICc0ehwIUfDTdwCRaMhsUKsz35zyYHDShrk/edit#slide=id.p